copy

```diff
@@ -0,0 +1,67 @@
+commit 3f6c80ad63bba52b1450651c298eb483c4fe2a15
```

Comment:
Hm, committing a patch is a bit unconventional. Is it possible you did `git add` instead of `git am` here?